pkg/ddc/alluxio/master.go

```diff
@@ -113,7 +113,9 @@ func (e *AlluxioEngine) ShouldSetupMaster() (should bool, err error) {
 	return
 }
 
-// setup the cache master
+// SetupMaster setup the master and Update the status of the runtime 
```

Comment:
setup the master and Update the status -> setups the master and updates the status